runtime/src/main/kotlin/kotlin/native/ForeignException.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package kotlin.native
+
+import kotlin.RuntimeException
+
+public class ForeignException internal constructor(val nativeException: Any?)  : RuntimeException()
```

Comment:
1. Why `RuntimeException`?
2. It makes sense to add some additional info to `ForeignException` from `nativeException`, e.g. the message.